src/js/helpers/searchIndexBuilder.js

```diff
@@ -29,7 +30,7 @@ const searchIndexBuilder = function(pages) {
             title: page.node[pageType].title,
             janisUrls: page.node.janisUrls,
             pageType: page.node.pageType,
-            summary: page.node[pageType].mission || page.node[pageType].shortDescription || ""
+            summary: page.node[pageType].mission || page.node[pageType].shortDescription || page.node[pageType].summary || ""
```

Comment:
on the joplin schema.py side of things, summary is misspelled as summery. I can fix it in a PR im gonna do soon?

Reply:
There's a "summary" field for Official Document Collections, and then there's an unimplemented "summery" field on the Joplin api schema. That "summery" field isn't used and could be removed.

Reply:
I can confirm that the summary field for the page types with it do have the data within the `searchIndex` on the search *component*. If you log the `searchIndex` in the */search* component, you'll see the summary fields are populated in the officialDocumentPages. Of course, you'll need to be on this branch to see it.